phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,14 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-queryserver-loadbalancer</artifactId>
```

Comment:
If the interface is defined in phoenix-queryserver moduleand implementation in phoenix-Load-balancer module, will it not make loadbalancer module depend on phoenix-queryserver module for compile time ( via maven). Which bring us back to same issue ?